js/screens/WebViewScreen.js

```diff
@@ -260,6 +271,16 @@ class WebViewScreen extends React.Component {
     this.keyboardWillShow?.remove();
     this.keyboardWillHide?.remove();
     this.siteManager.refreshSites();
+    this.appStateSubscription?.remove();
+  }
+
+  _sendAppStateChange(appState) {
+    const appStateChange = `
+      window.dispatchEvent(new CustomEvent("AppStateChange", { newAppState: "${appState}" }));
```

Comment:
I went with this for now @markvanlan, let me know how the core integration goes.